fvm/errors/errors.go

```diff
@@ -114,6 +114,39 @@ func HandleRuntimeError(err error) error {
 		return NewUnknownFailure(err)
 	}
 
+	errs := []error{runErr}
+	var parentErr errors.ParentError
+	if As(err, &parentErr) {
+		errs = parentErr.ChildErrors()
```

Comment:
we could wrap `errors.ParentError` in another struct that implements an `Unwrap() []errors` method, however, `errors.As` will return the first error in the list that matches. If there are multiple, and one is a failure, it could be hidden.